src/dataflow/components/nodes/controls/sensor-value-control.tsx

```diff
@@ -3,6 +3,7 @@ import Rete, { NodeEditor, Node } from "rete";
 import { NodeSensorTypes } from "../../../utilities/node";
 import "./sensor-value-control.sass";
 import "./value-control.sass";
+import { kEmptyValueString } from "../factories/dataflow-rete-node-factory";
```

Comment:
nit: by convention we've generally been keeping the SASS imports last, which is made more difficult by the fact that VSCode's auto-import mechanism always adds new imports last.

Reply:
:+1:

Reply:
:+1: